pg_search/src/api.rs

```diff
@@ -22,9 +22,88 @@ pub fn weighted_mean(a: f64, b: f64, weights: Vec<f64>) -> f64 {
     );
 
     assert!(
-        (weight_a + weight_b - 1.0).abs() < std::f64::EPSILON,
+        (weight_a + weight_b - 1.0).abs() < f64::EPSILON,
         "Weights must add up to 1"
     );
 
     a * weight_a + b * weight_b
 }
+
+#[cfg(test)]
```

Comment:
there should only be one mod, called `mod tests`. You should have only #[pg_test], not `#[test]` in it.